src/components/Tasks/TasksBox.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Box, GU, useViewport } from '@aragon/ui'
 import TaskAmounts from './TasksAmounts'
 
-const TasksBox = () => {
+function TasksBox() {
```

Comment:
ohh sorry about that, you try to keep using the function prefix as i can see

Reply:
Ah no worries it’s just a habit from the other Aragon codebases (where we tend to use normal functions for top level components), but I don’t mind either way. Let me know what you prefer and I’ll follow :-)

Reply:
I think that makes more sense to see there the "function" for the top level component, i am good with that

Reply:
Cool, let’s do that then :smile: